_chapters/chapter_01.md

```diff
@@ -3,6 +3,7 @@ url: 1.html
 number: 1
 title: Getting Started from Scratch
 layout: rails_tutorial
+chapter: 1
```

Comment:
I think this `chapter` data attribute might be redundant to the `number` data attribute.